browser/src/view/components/Button/Button.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './Button.scss';
+
+export const Button = ({ layout, label, Icon, type }) => {
+  return (
+    <button
+      className={`${
+        type === ButtonAttributes.Type.destructive
+          ? 'btn-destructive'
+          : type === ButtonAttributes.Type.primary
+          ? 'btn-primary'
+          : 'btn-secondary'
+      }
```

Comment:
Something like this should be a function/map.

```js
const getButtonClassnameByType = type => ({
  [Button.TYPE.DESTRUCTIVE]: 'destructive',
  [Button.TYPE.PRIMARY]: 'primary',
  ...etc
})[type] || Button.TYPE.PRIMARY;
```

Reply:
Ooooh, thank you. I knew there was a cleaner way to do this. Is this particular syntax a different way of calling .map()?

Reply:
Nope, `.map` and a `map` are two different things.  

`.map` is philosophically "mapping" one value to another.

A `map` is more like a dictionary whose keys can be used to map to specific values.